README.md

```diff
@@ -64,6 +65,29 @@ The following data is required to configure the storefront to communicate with y
 
 Visit [Kibo documentation](https://apidocs.kibong-perf.com/?spec=graphql#auth) for more details on API authentication
 
+next.config.js example
+
+```bash
+hubName= hub_name
```

Comment:
This is confusing, is this code that needs to be set in environment variables? it says next.config.js
How would a dev configure this for their own amplience?

Reply:
We don't required environment variables for amplience cms. Using 'npm install dc-delivery-sdk-js' sdk, we need to pass hubName and 'schema url' parameters to ContentClient(class from SDK) to fetch data from amplience cms, that's why we are passing hubName and schema URL in next.config.js file.